daffodil-test/src/test/scala/org/apache/daffodil/section00/general/TestFileBuffering.scala

```diff
@@ -33,36 +33,36 @@ class TestUnparserFileBuffering {
 
   import TestUnparserFileBuffering._
 
-  @Test def test_puaInfosetChars_01(): Unit = { runner.runOneTest("puaInfosetChars_01") }
-  @Test def test_puaInfosetChars_02(): Unit = { runner.runOneTest("puaInfosetChars_02") }
+  @Test def test_puaInfosetChars_01(): Unit = { runner.runOneTest("puaInfosetChars_01_ffb") }
```

Comment:
Do we want to add the _ffb suffix to the JUnit testname as well?

Reply:
Good idea. Will do.